drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -336,12 +336,23 @@ def alerts
       load_ar_association(object, :alerts, scope: Hmis::ClientAlert.active).sort_by(&:created_at).reverse
     end
 
+    # not optimized for batch queries, causes n+1 queries
     def hud_chronic
       return unless current_permission?(permission: :can_view_hud_chronic_status, entity: object)
 
-      # causes n+1 queries
-      enrollments = object.enrollments.where(data_source_id: current_user.hmis_data_source_id)
-      !!object.destination_client&.as_warehouse&.hud_chronic?(scope: enrollments)
+      open_enrollments = object.enrollments.open_excluding_wip.preload(:ch_enrollment, :project)
+
+      # Check if there's an open enrollment at a residential project with a move in date that has occurred
+      today = Date.current
+      client_is_housed = open_enrollments.any? do |enrollment|
+        enrollment.project.project_type.in?(HudUtility2024.residential_project_type_ids) && (enrollment.move_in_date&.<= today)
+      end
+      return false if client_is_housed
+
+      # Check chronic status at open enrollments for homeless project types
+      open_enrollments.any? do |enrollment|
+        enrollment.project.project_type.in?(HudUtility2024.chronic_project_types) && enrollment.ch_enrollment&.chronically_homeless?
```

Comment:
maybe chronic_project_types should be homeless_project_types which includes transitional housing

Reply:
I don't think so, the reporting glossary calls out these 4 for chronic-at-point in time.
<img width="913" alt="Screenshot 2024-11-25 at 7 31 12 PM" src="https://github.com/user-attachments/assets/52ee269a-d886-4ff5-8635-5ad7dc72f034">